src/controllers/expense.controller.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-console */
+'use strict';
+
+const expenseService = require('../services/expense.service');
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  let filteredExpenses = expenseService.getAll();
+
+  if (userId) {
+    filteredExpenses = expenseService.getAllByUserId(+userId, filteredExpenses);
+  }
+
+  if (categories) {
+    filteredExpenses = expenseService
+      .getAllByCategory(categories, filteredExpenses);
+  }
+
+  if (from) {
+    filteredExpenses = expenseService.getAllByDateFrom(from, filteredExpenses);
+  }
+
+  if (to) {
+    filteredExpenses = expenseService.getAllByDateTo(to, filteredExpenses);
+  }
+
```

Comment:
You don't need to create functions for these operations in services
You can do this logic right here